stake-pool/single-pool-cli/src/quarantine.rs

```diff
@@ -0,0 +1,197 @@
+use {
+    base64::{prelude::BASE64_STANDARD, Engine},
+    solana_sdk::{
+        instruction::Instruction,
+        native_token::LAMPORTS_PER_SOL,
+        pubkey::Pubkey,
+        signature::{Keypair, Signer},
+        stake::{
+            self,
+            state::{Authorized, Lockup, Meta, Stake, StakeState},
+        },
+        system_instruction, system_program,
+        sysvar::{self, rent::Rent},
+        transaction::Transaction,
+    },
+    solana_vote_program::{
+        vote_instruction::{self, CreateVoteAccountConfig},
+        vote_state::{VoteInit, VoteState, VoteStateVersions},
+    },
+};
+
+use crate::config::*;
+
+// XXX this file is stuff that should not be in the cli
+// once i have a better understanding of needs/affordances im making a stake program client
+
+pub async fn get_rent(config: &Config) -> Result<Rent, Error> {
+    let rent_data = config
+        .program_client
+        .get_account(sysvar::rent::id())
+        .await?
+        .unwrap();
+    let rent = bincode::deserialize::<Rent>(&rent_data.data)?;
+
+    Ok(rent)
+}
+
+pub async fn get_minimum_delegation(config: &Config) -> Result<u64, Error> {
```

Comment:
Ah, this is kinda annoying... but I think you're right, this is the best option with a `ProgramClient`. Much better than adding to that trait

Reply:
thanks to doing a code review i learned today theres an rpc call for this in 1.14 so im gonna use that instead. this is gonna be abstracted when i write a stake program client anyway

Reply:
Ah my bad, I forgot that you had an explicit rpc client. I thought everything was going through `ProgramClient`.  In that case, definitely use it!